dev/docker/python/Dockerfile

```diff
@@ -4,10 +4,13 @@ RUN apt-get update && apt-get install -y r-base && Rscript -e "install.packages(
 
 WORKDIR /usr/src/app
 
-COPY repos repos
-COPY repos/delphi/delphi-epidata/dev/docker/python/setup.sh .
-
 RUN ln -s -f /usr/share/zoneinfo/America/New_York /etc/localtime && \
-    chmod -R o+r repos/ && \
-    bash setup.sh && \
-    pip install --no-cache-dir -r repos/delphi/delphi-epidata/requirements.api.txt -r repos/delphi/delphi-epidata/requirements.dev.txt
+    mkdir delphi && \
+    chmod -R o+r delphi/
+
+COPY repos/delphi/operations/src delphi/operations
+COPY repos/delphi/utils/src delphi/utils
+COPY repos/delphi/delphi-epidata/src delphi/epidata
```

Comment:
Comment (non-blocking): this COPY command is later overwritten by a runtime mount between the same folders we run tests. Consider getting rid of this.

Thinking out loud (non-blocking): looks like we're dropping the top-level `repos` dir within the Dockerfile. Want to make sure we think through the consequences of this. AFAICT, previously we copied the entire local `repos` folder over int `/usr/src/app/repos` and then moved pieces over to `/usr/src/app/delphi/` and `/usr/src/app/undefx/`. So it looks like we're skipping the intermediate step. So is there anything that still relies on that `repos` folder? The only thing I can find are the Docker testing commands that end with e.g. `python -m pytest --import-mode importlib repos/delphi/delphi-epidata/integrations`. But that is handled by the first mount command in those Docker commands: 

```sh
--mount type=bind,source=./repos/delphi/delphi-epidata,target=/usr/src/app/repos/delphi/delphi-epidata,readonly \
```

So the edit to my first sentence from this sequence of thoughts is: we're not dropping the `repos` dir, it just gets added by a mount at runtime.

Comment (non-blocking, separate issue): it's tough to mentally imagine the directory structure of Dockerfile when runtime mounts are involved (due to non-locality between Docker run command and the Dockerfile). Maybe we should add a comment at the end of this Dockerfile saying something like "in practice, when we run this, these additional directories are present". 

Comment (non-blocking, separate issue): a future future solution to this non-locality is to make a unified `docker-compose.yml` file.

Reply:
Docker compose PoC: #1315